posts/hello-world-in-kafka-using-python.md

```diff
@@ -19,30 +19,36 @@ You might have got a hint of it already. This blog is about understanding what i
 ## What is Kafka? Why should one use it?
 In short, Kafka is a distributed streaming and message queueing platform.
 
-> Oh wait! What does that even mean? How different is it from traditional databases? 
+> Oh wait! What does that even mean?
 
+A streaming platform is a system that can perform the following:
+1. Store and process streams of data in real time
+2. Allow applications and/or services to publish and subscribe to these data streams at scale
 
-Simplified explanation of what are streaming and messaging queues and how they are different from traiditional databases.
+> Interesting! How different or similar is it from traditional databases?
 
+Kafka is NOT a database. Nevertheless, it has few similarities and quite an amount of differences with respect to traditional databases.
```

Comment:
Don't give any space for a beginners imagination to run wild. Just distance Kafka from other databases, just say Kafka is really different from traditional databases.